### DISCUSSION TITLE
Update UI to adopt 3 elements from Bitwarden's 2023 desktop app user-interface

### DISCUSSION THREAD

**Discussion by gmejzej** [keepassxreboot/keepassxc#9411] [Ideas]
I am a big KeePassXC fan. But the KeePassXC UI has problems, which were revealed to me after I gave Bitwarden a try. Bitwarden has 3 excellent UI elements that I wish KeePassXC could adopt:

1) Bitwarden has a Vertical Column View: 3 columns of information (group:item:content). This allows the user to click on a group and then an item and then see the content for that item (including notes and custom fields) all on the same screen with no popups. Although KeePassXC has a horizontal view option, this is inferior as it doesn't display all the information and does not offer the same simplicity of interaction and display.

2) Bitwarden has an "Edit" button when landing on an item. When I click an item, the content displayed in Bitwarden cannot be changed until the "Edit" button is clicked. This is an excellent safety mechanism to prevent unintended changes to content. Additionally, Bitwarden's content can be copied from the non-edit view. In KeePassXC, I am CONSTANTLY interacting with KeePassXC notes, copying something, and then closing the popup results in a save dialogue. Did I accidentally change something? Should I click save and potentially save an unintended change? This problem does not exist in Bitwarden cause of the "edit" button.

3) Bitwarden displays "Custom Fields" and gives options like text, hidden, binary, etc. These are great for storing PINs, security questions, recovery keys, and secondary passwords for functions set within a larger account. Although KeePassXC has the Advanced tab, it is cumbersome to get to, has no default easy copy function, and overall is inferior to Bitwarden's UI setup.

Can KeePassXC please adopt the above UI features?

**Comment by Opicka879** [comment]
Upvote from me. Point 1 is the only reason im not using KeePassXC and still using KeePass with Plugins. There should be third column instead of the section below the list of records. There should be all entries, like General and Advanced tab and TOTP together with option of add/edit/copy/unhide entry.
